### DISCUSSION TITLE
License is no longer accepted

### DISCUSSION THREAD

**Discussion by Nokthem** [aclap-dev/video-downloadhelper#1482] [License & Purchase requests]
### What's your issue?

Hello,
I had to delete and reinstall Firefox today.
When I copied my license into the download helper, it was suddenly no longer accepted...
Can you help me?

### The 10 first digits (only 10) of your license number or email address

63919da905

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry the 10 digits you gave do not match with any license.

**Reply by Nokthem** [reply]
and at my email address?
Nokthem@web.de
I had bought a second one at some point

**Reply by jcfrog** [reply]
you have 2 licenses, both are valid. 
I sent you an email.
